lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1368,7 +1395,12 @@ SPIRVValue *LLVMToSPIRV::transValueWithoutDecoration(Value *V,
     if (ST->isAtomic())
       return transAtomicStore(ST, BB);
 
-    std::vector<SPIRVWord> MemoryAccess(1, 0);
+    // Keep this vector to store MemoryAccess operands for both Alignment and
```

Comment:
MemoryAccess fullfilling is the same for load and store intructions. May it be moved into a separate function?